### PR TITLE
l10n: Spelling unification

### PR DESCRIPTION
**Pull request by Valdnet**
Spelling unification in Transifex.

Signed-off-by: Valdnet <47037905+Valdnet@users.noreply.github.com>

### CODE REVIEW

File: src/AdminSettings.vue [hunk 1]
```diff
@@ -30,9 +30,9 @@
 					class="checkbox"
 					:disabled="loading"
 					@change="saveData">
-				<label for="admin_approval">{{ t('registration', 'Require admin approval') }}</label>
+				<label for="admin_approval">{{ t('registration', 'Require administrator approval') }}</label>
```
Comment (nickvergessen) [inline src/AdminSettings.vue:33]: Should we use "administration" like in the readme and info.xml?

Comment (Valdnet) [reply src/AdminSettings.vue:33]: Yes you are right 😉.

**Review by nickvergessen — APPROVED** [review]

**Comment by nickvergessen** [comment]
@Valdnet should we really do this? In lots of other places across Nextcloud it's always "administrator" (5.172 results) and administration only has 42 (basically only 2 texts in encryption)

**Comment by Valdnet** [comment]
@nickvergessen I definitely need to change the following text strings:
`Require admin approval`

`Admins will be notified via email for new user creation or require approval`

`Enabling "admin approval" will prevent registrations from mobile and desktop clients to complete as the credentials cannot be verified by the client until the user was enabled.`

Because we do not use the word "Admin" in the Nextcloud instance. By the way, I changed the remaining strings to be consistent.

**Comment by Valdnet** [comment]
Well. I'm for it too 😉.

**Comment by nickvergessen** [comment]
Please stop modifying one file at a time. I will fix it

**Comment by Valdnet** [comment]
I read your comment too late. I've already done it. Sorry.

**Pull request merged**